cirq_superstaq/custom_gates.py

```diff
@@ -219,43 +239,49 @@ def _decompose_(self, qubits: Tuple[cirq.LineQubit, cirq.LineQubit]) -> cirq.OP_
             *qubits
         ) ** -0.25
         yield cirq.X(qubits[0])
+        if self.sandwich_rx_rads is not None:
+            yield cirq.rx(self.sandwich_rx_rads)(qubits[1])
         yield cirq_superstaq.CR(*qubits) ** -0.25 if self.polarity == "+-" else cirq_superstaq.CR(
             *qubits
         ) ** 0.25
 
     def _circuit_diagram_info_(
         self, args: cirq.CircuitDiagramInfoArgs
     ) -> cirq.protocols.CircuitDiagramInfo:
-        return cirq.protocols.CircuitDiagramInfo(
-            wire_symbols=(f"AceCR{self.polarity}(Z side)", f"AceCR{self.polarity}(X side)")
-        )
+        top, bottom = f"AceCR{self.polarity}(Z side)", f"AceCR{self.polarity}(X side)"
+        if self.sandwich_rx_rads is not None:
+            bottom += f"|{cirq.rx(self.sandwich_rx_rads)}|"
+        return cirq.protocols.CircuitDiagramInfo(wire_symbols=(top, bottom))
 
     def _qasm_(self, args: cirq.QasmArgs, qubits: Tuple[cirq.Qid, cirq.Qid]) -> Optional[str]:
-        """QASM symbol for AceCR('+-') (AceCR('-+')) is acecr_pm (acecr_mp)"""
-        polarity_str = self.polarity.replace("+", "p").replace("-", "m")
-        return args.format(
-            "acecr_{0} {1},{2};\n",
-            polarity_str,
-            qubits[0],
-            qubits[1],
-        )
+        """QASM symbol for AceCR("+-") (AceCR("-+")) is acecr_pm (acecr_mp)
 
-    def __eq__(self, other: object) -> bool:
-        if not isinstance(other, AceCR):
-            return False
-        return self.polarity == other.polarity
+        If there is a sandwich, it comes last. For example, AceCR("-+", np.pi / 2) has qasm
+        acecr_mp_rx(pi*0.5).
+        """
+        polarity_str = self.polarity.replace("+", "p").replace("-", "m")
+        gate_name_qasm = f"acecr_{polarity_str}"
+        kwargs: dict = {"q0": qubits[0], "q1": qubits[1]}
+        if self.sandwich_rx_rads is not None:
+            kwargs["theta"] = self.sandwich_rx_rads / np.pi
+            gate_name_qasm += "_rx({theta:half_turns})"
+        return args.format(gate_name_qasm + " {q0},{q1};\n", **kwargs)
```

Comment:
can be simplified a bit:
```suggestion
        if not self.sandwich_rx_rads:
            return args.format("acecr_{} {},{};\n", polarity_str, *qubits)
        exponent = self.sandwich_rx_rads / np.pi
        return args.format("acecr_{}_rx({:half_turns}) {},{};\n", polarity_str, exponent, *qubits)
```